dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details_facilitator.html.haml

```diff
@@ -36,6 +36,6 @@
 %p
   Thank you,
   %br
-  Dave Frye
+  Sam Morris
```

Comment:
When is this one sent? Based on the name is seems like its sent to facilitators is that right? If so it should come from Andrea

Reply:
It's sent in the `teacher_enrollment_receipt` and `teacher_enrollment_reminder` emails and in both emails, only when `Pd::Workshop::COURSE_FACILITATOR == @workshop.course`. So I think it is to facilitators

Reply:
Ok then lets have it come from Andrea. Thank you!